src/App.tsx

```diff
@@ -27,6 +27,14 @@ export const App = () => (
   </Nest>
 );
 
+// Helper component, for wrapping the app's providers around stories and tests
+export const MockApp: FC<{ children: ReactNode }> = ({ children }) => (
+  <Nest elements={providers}>
+    <CssBaseline />
+    {children}
+  </Nest>
+);
```

Comment:
I have a few thoughts I'll put all here:
- We probably need always display the storybook components in context of our theme. Otherwise there's going to be a disconnect.
- It would be great to have that automatically applied everywhere.
- Conversely, I don't think we want the other providers initialized. All of these components for storybook should be context free, right? So they don't need any app-level context providers.

Reply:
Your last point about not needing providers is a good catch. Now that I'm playing around with it, I'm realizing I didn't need this wrapper at all. I think this was a classic case of me pre-emptively adding complexity where it wasn't needed. I'm going to eliminate the `MockApp` component for now

Reply:
> I think this was a classic case of me pre-emptively adding complexity where it wasn't needed.

I NEVER do that 😝 

Sounds good 👍

Reply:
😆